### ISSUE TITLE
MAS2.4.3 Violation: Focus order is illogical when NavigationView in Left DisplayMode

### ISSUE THREAD

**Issue by jskollin** [Microsoft/microsoft-ui-xaml#519]
**Describe the bug**
Violation of MAS Rule 2.4.3 - Focus Order

When the NavigationView is in the Left DisplayMode, the top NavigationViewItems and the NavigationViewItems in the Footer are not only displayed as two separate lists, but are read by the narrator as two separate lists. Because of this, users would assume that continually hitting the down arrow from the top NavigationViewItems would eventually hit the end of the list, and prevent the user from navigating down to the Footer items.

**Steps to reproduce the bug**
This can be seen using the _API in action_ section of the NavigationView page in the XAML Controls Gallery application. If you select the option the enables the PaneFooter to be visible, you can traverse through all the items in the NavigationView with the up and down arrow keys.  

**Expected behavior**
The behavior should be that the down arrow does not permit the user to jump from the top NavigationViewItems to the Footer items. Using the NavigationView from _API in action_ section, users should not be able to use the down arrow to navigate from _MenuItem3_ to top item in the Footer.

**Comment by jevansaks** [comment]
To a keyboard only user, the current behavior is desirable. But because the main list and the footer list are separate, it might not be clear to a narrator user. 

That MAS rule says that focus order should be logical, and it seems like that's the case. 

This might not be the perfect narrator experience, but it seems ok. Certainly doesn't seem like a MAS focus order violation to us.
